contracts/PWNVault.sol

```diff
@@ -92,19 +92,14 @@ contract PWNVault is Ownable, IERC1155Receiver {
      * To accept the transfer, this must return
      * `bytes4(keccak256("onERC1155Received(address,address,uint256,uint256,bytes)"))`
      * (i.e. 0xf23a6e61, or its own function selector).
-     * @param operator The address which initiated the transfer (i.e. msg.sender)
-     * @param from The address which previously owned the token
-     * @param id The ID of the token being transferred
-     * @param value The amount of tokens being transferred
-     * @param data Additional data with no specified format
      * @return `bytes4(keccak256("onERC1155Received(address,address,uint256,uint256,bytes)"))` if transfer is allowed
      */
     function onERC1155Received(
-        address operator,
-        address from,
-        uint256 id,
-        uint256 value,
-        bytes calldata data
+        address /*operator*/,
```

Comment:
is that just space saving?

Reply:
this silences a warning about unused variable